src/main/webapp/app/exercises/quiz/manage/quiz-exercise.component.ts

```diff
@@ -225,8 +227,29 @@ export class QuizExerciseComponent extends ExerciseComponent {
         );
     }
 
+    public sortRows() {
+        if (this.predicate === 'statusAsNumber') {
+            this.quizExercises.forEach((a) => {
+                a.statusAsNumber = this.statusForQuiz(a);
+            });
+        }
+
+        this.sortService.sortByProperty(this.quizExercises, this.predicate, this.reverse);
+    }
+
     /**
-     * Do nothing
+     * Returns the status of the quiz represented as numbers, whether the quiz is
+     * open for practice or is it visible before it starts.
+     * @param quizExercise The quizExercise object.
      */
-    callback() {}
+    private statusForQuiz(quizExercise: any) {
+        if (quizExercise.isPlannedToStart && quizExercise.remainingTime != null) {
+            if (quizExercise.remainingTime <= 0) {
+                return quizExercise.isOpenForPractice ? 1 : 0;
```

Comment:
The numbers seem a bit arbitrary here. Not sure if storing them in the quiz exercise object really makes sense.
Would it make sense to define a custom sort method instead?

Reply:
addressed via ea8ea78